internal/cleanup/cleanup_test.go

```diff
@@ -49,8 +49,6 @@ func TestNewExposureHandler(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		tc := tc
```

Comment:
You still need this. Otherwise all tests will run as the last entry in `testCases` because of how `t.Parallel()` works.

Reply:
Ah thanks. Sorry was just poking at something that looked funny to me.